portal/types/stakingDashboard.ts

```diff
@@ -49,6 +43,18 @@ export type StakingPosition = {
   transferable: boolean
 }
 
+export type StakingDashboardOperation = {
+  approvalTxHash?: Hash
+  input?: string
+  inputDays?: string
+  lockupDays?: number
+  transactionHash?: Hash
+  stakingPosition?: Partial<
+    Pick<StakingPosition, 'amount' | 'tokenId' | 'lockTime'>
+  >
+  status?: StakingDashboardStatus
+}
```

Comment:
A minor suggestion, but if you expect everything to be optional (particularly, if you expect it when adding properties in the future), you can use `Partial`
```suggestion
export type StakingDashboardOperation = Partial<{
  approvalTxHash: Hash
  input: string
  inputDays: string
  lockupDays: number
  transactionHash: Hash
  stakingPosition: Partial<
    Pick<StakingPosition, 'amount' | 'tokenId' | 'lockTime'>
  >
  status: StakingDashboardStatus
}>
```

Is every field optional for a reason?